src/test/librados/snapshots.cc

```diff
@@ -157,15 +157,13 @@ TEST_F(LibRadosSnapshotsPP, SnapCreateRemovePP) {
   ASSERT_EQ(0, ioctx.remove("foo"));
   ASSERT_EQ(0, ioctx.snap_create("snapbar"));
 
-  librados::ObjectWriteOperation *op = new librados::ObjectWriteOperation();
+  std::unique_ptr<librados::ObjectWriteOperation> op(new librados::ObjectWriteOperation());
```

Comment:
@tchaikov I can't find any examples existing in the code base?

$ ag --cpp --ignore ./src/boost/\* "make_unique"
$